### PR TITLE
fix(nasha.partition): no more partition name edit

### PR DESCRIPTION
**Pull request by y4nnL**
| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/nasha-refresh-deletions`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #MANAGER-9721
| License          | BSD 3-Clause

- [X] Try to keep pull requests small so they can be easily reviewed.
- [X] Commits are signed-off
- [X] Only FR translations have been updated
- [X] Branch is up-to-date with target branch
- [X] Lint has passed locally
- [X] Standalone app was ran and tested locally
- [X] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~~Breaking change is mentioned in relevant commits~~

### CODE REVIEW

File: packages/manager/modules/nasha/src/dashboard/partition/partition.routing.js [hunk 1]
```diff
@@ -4,24 +4,21 @@ export default /* @ngInject */ ($stateProvider) => {
...
   const taskTrackerStateName = `${stateName}.task-tracker`;
 
-  const goToEditResolve = ['description', 'name', 'size'].reduce(
-    (resolves, id) => {
-      const capitalizedId = `${id[0].toUpperCase()}${id.slice(1)}`;
-      return {
-        ...resolves,
-        [`goToEdit${capitalizedId}`]: /* @ngInject */ (
-          $state,
+  const goToEditResolve = ['description', 'size'].reduce((resolves, id) => {
```
Comment (marie-j) [inline packages/manager/modules/nasha/src/dashboard/partition/partition.routing.js:7]: This looks highly complicated for the wanted result

File: packages/manager/modules/nasha/src/dashboard/partition/partition.template.html [hunk 2]
```diff
@@ -3,25 +3,14 @@
...
-                    aria-hidden="true"
-                ></span>
-            </button>
         </div>
         <p data-ng-bind="$ctrl.nasha.serviceName"></p>
         <oui-header-tabs>
             <oui-header-tabs-item
                 data-href="{{:: $ctrl.partitionHref() }}"
-                data-active="[$ctrl.partitionHref(), $ctrl.editNameHref(), $ctrl.editDescriptionHref(), $ctrl.editSizeHref()].includes($ctrl.currentHref())"
+                data-active="[$ctrl.partitionHref(), $ctrl.editDescriptionHref(), $ctrl.editSizeHref()].includes($ctrl.currentHref())"
```
Comment (marie-j) [inline packages/manager/modules/nasha/src/dashboard/partition/partition.template.html:13]: We should avoid this kind of logic in the HTML (especially polyfills won't be applied)

**Review by marie-j — CHANGES_REQUESTED** [review]

File: packages/manager/modules/nasha/src/dashboard/partition/partition.routing.js (hunk 1 above)
Comment (y4nnL) [reply packages/manager/modules/nasha/src/dashboard/partition/partition.routing.js]: :+1:

File: packages/manager/modules/nasha/src/dashboard/partition/partition.template.html (hunk 2 above)
Comment (y4nnL) [reply packages/manager/modules/nasha/src/dashboard/partition/partition.template.html]: :+1: 
Moved into the routing

**Review by marie-j — APPROVED** [review]

**Review by ghyenne — APPROVED** [review]

**Pull request merged**